### DISCUSSION TITLE
Beatmap backgrounds in song select get stretched by transforms

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#15499] [Q&A] [answered]
I just noticed that the backgrounds in the carousel card things (and the bigger background preview on the top left) get stretched or squished whenever you do things like opening the settings or notifications, scrolling up and down, changing beatmap or clicking the cards themselves. The only action that properly crops the backgrounds seems to be changing screen resolution.

Quick demo below:

https://user-images.githubusercontent.com/36758269/140610216-ceac03a9-da6f-4867-b891-5db527a9d697.mp4

This squishiness is more pronounced when panels go off-screen, because the background itself is clamped to the screen:

https://user-images.githubusercontent.com/36758269/140610220-34ad22d4-9276-499d-8088-ed6dfb1eee79.mp4

And here is a cropped view of the top-left preview panel:

https://user-images.githubusercontent.com/36758269/140610527-e4b9a052-a40c-43b0-9e6f-c9b456dc125a.mp4

Is this intended design-wise? To me it looks pretty weird once you notice it and I can't unsee it

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15475

will be fixed next release
